content/en/tracing/setup/dotnet.md

```diff
@@ -67,23 +67,17 @@ Install the .NET Tracer on the host using the [MSI installer for Windows][1]. Ch
 
 Install the .NET Tracer on the host using the [MSI installer for Windows][1]. Choose the platform that matches the OS architecture.
 
-Add the `Datadog.Trace.ClrProfiler.Managed` [NuGet package][2] to your application, matching the package version to the MSI installer above. Refer to the [NuGet documentation][3] for instructions on how to add a NuGet package to your application.
-
 - Native library: deployed into `Program Files` by default and registered as a COM library in the Windows Registry by the MSI installer.
-- Managed libraries: deployed together with your application when it is published (via NuGet package).
-- Environment variables: added for IIS only by the MSI installer. Applications that do not run in IIS need [additional configuration][4] to set these environment variables.
+- Managed libraries: deployed together with your application when it is published.
```

Comment:
This is not correct anymore. The managed library is deployed with the native library (in the msi installer)